components/Poll/Poll.jsx

```diff
@@ -0,0 +1,407 @@
+        text: optionText,
+        usersVoted: [],
+      });
+      this.props.savePoll({
+        title: this.state.title,
+        multiSelect: this.state.multiSelect,
+        open: this.state.open,
+        options: opts,
+      });
+      this.setState({ newOption: '' });
+    }
+  }
+
+  buildListItems = () => {
+    if (!this.state.options) {
+      return;
+    }
+    let items = this.state.options
+            .sort((a, b) => b.usersVoted.length - a.usersVoted.length)
+            .slice()
+            .map(item => {
+              if (item._id) {
+                item.key = item._id;
+              }
+              return item;
+            });
+    if (!!this.props.isAuthor || this.state.open) {
+      items.push({ key: 'add_option_item' });
+    }
+    return items;
+  }
+
+  renderListItem = ({ item }) => {
+    if (item.key === 'add_option_item') {
+      return (
+        <View style={styles.optionGroup}>
+          <TouchableOpacity style={styles.addContainingView} onPress={this.addOption}>
+            <Icon name="add" style={styles.addView}/>
+          </TouchableOpacity>
+          <Textarea
+            placeholder="Add an option..."
+            style={styles.addOptionText}
+            onChangeText={this.newOptionUpdate}
+            value={this.state.newOption}
+          />
+        </View>
+      );
+    }
+    let selected = (this.userVoteIndex(item) !== -1);
+    let totalCount = this.getTotal(this.state.options);
+    let percentage = totalCount === 0 ? 0 : (item.usersVoted.length / totalCount * 100).toFixed();
+    let canDelete = this.props.isAuthor || this.state.isAdmin || item.creator === this.props.loggedInUser._id;
+    return(
+  		<TouchableOpacity style={styles.fillWidth} onPress={this.toggleOption.bind(this, item)} onLongPress={this.confirmDelete.bind(this, item)}>
+        <View style={styles.optionView}>
+          {this.state.multiSelect ?
+            <CheckBox
+              style={styles.optionButton}
+              containerStyle={styles.optionButtonContainer}
+              checked={selected}
+              onIconPress={this.toggleOption.bind(this, item)}
+              onLongIconPress={this.confirmDelete.bind(this, item)}
+            /> :
+            <CheckBox
+              style={styles.optionButton}
+              containerStyle={styles.optionButtonContainer}
+              checkedIcon='dot-circle-o'
+              uncheckedIcon='circle-o'
+              checked={selected}
+              onIconPress={this.toggleOption.bind(this, item)}
+              onLongIconPress={this.confirmDelete.bind(this, item)}
+            />}
+  			  <Autolink style={styles.optionText} text={item.text}/>
+  			  <Text style={styles.optionCount}>{`${item.usersVoted.length}: ${percentage}%`}</Text>
+          {canDelete ?
+            <TouchableOpacity style={styles.optionDeleteContainer} onPress={this.confirmDelete.bind(this, item)}>
+              <Icon
+                name="delete"
```

Comment:
Not quite centered vertically, it's a little lower than the rest of the line

Reply:
Should be fixed in most recent update, please tell me if it looks good to you